crates/collab_ui/src/collab_titlebar_item.rs

```diff
@@ -730,8 +730,8 @@ impl CollabTitlebarItem {
                 .menu(|cx| {
                     ContextMenu::build(cx, |menu, _| {
                         menu.action("Settings", zed_actions::OpenSettings.boxed_clone())
-                            .action("Extensions", extensions_ui::Extensions.boxed_clone())
-                            .action("Themes...", theme_selector::Toggle::default().boxed_clone())
+                            .action("Extensions…", extensions_ui::Extensions.boxed_clone())
```

Comment:
The "Extensions" menu is going to open a tab page, not a dialog. In the OS (Windows & macOS) UX, it is only the modal dialog need suffix with `...`.

Reply:
For example, the Chrome menu:

The `Downloads` menu will open a tab page, so it not have `...` suffix.

And the print is open a dialog, it has that.

![image](https://github.com/zed-industries/zed/assets/5518/23badc17-1a5c-4c15-8e66-da24d15c70bc)

Reply:
Nice catch. This is because in my head Extensions is already a modal, but I haven't actually done that work yet!

![CleanShot 2024-05-21 at 10 31 58@2x](https://github.com/zed-industries/zed/assets/1714999/5579fba6-922b-46b1-8c50-6c575fa62c41)

Reply:
I see